main.py

```diff
@@ -1,15 +1,34 @@
+alphabet = [chr(i) for i in range(32,127)]
+
 def encrypt(plain, key):
     # type: (str, int) -> str
 
     # TODO your code will be here
-    # this is new comment
+    encrypted_text = ""
+    key = key % len(alphabet)
+    for i in plain:
+        if key + alphabet.index(i) >= len(alphabet):
+            new_index = key + alphabet.index(i) - len(alphabet)
+        else:
+            new_index = key + alphabet.index(i)
+        encrypted_text += alphabet[new_index]
+    return encrypted_text
     pass
```

Comment:
`pass` is used as a placeholder inside loops, functions, class. So we don't need here.